### PR TITLE
Add moonraker-telegram-bot by nlef

### PR DESCRIPTION
**Pull request by shavrin777**
Hello th33xitus, 

This is a PR to add our telegram bot to your magnificient script. 
We have been polishing it for a while, and believe it is now in a state where almost anyone will profit from its installation.

This is a very good alternative to opening up your network, as in our experience a lot of people do, and a nice alternative to the web interface for notifications.
Also, timelapses! Good ones, not like the ones currently used by the people.

We have prepared everything, and in our opinion eveything should fit nicely.
Hopefully it does fit in in your opinion as well. 
If you have any questions, we can have a quick chat on discord/telegram, or simply in this PR.

Thank you in advance,
The moonraker-telegram-bot team.

### CODE REVIEW

File: scripts/backup.sh [hunk 1]
```diff
@@ -179,3 +179,17 @@ backup_klipperscreen(){
     ERROR_MSG=" Can't backup KlipperScreen directory!\n Not found!"
   fi
 }
+
+backup_MoonrakerTelegramBot){
```
Comment (th33xitus) [inline scripts/backup.sh:183]: it must be called `backup_MoonrakerTelegramBot(){`

Comment (shavrin777) [reply scripts/backup.sh:183]: ooops

Comment (shavrin777) [reply scripts/backup.sh]: oops x2 :)

File: scripts/ui/remove_menu.sh [hunk 2]
```diff
@@ -6,7 +6,6 @@ remove_ui(){
   echo -e "|  --> Your printer configuration directory             | "
   echo -e "|  --> ~/kiauh-backups                                  | "
   echo -e "|  You need remove them manually if you wish so.        | "
-  hr
```
Comment (th33xitus) [inline scripts/ui/remove_menu.sh:9]: This `hr` needs to be added again.

**Pull request merged**